src/server/rdb_save.cc

```diff
@@ -1171,11 +1171,16 @@ class RdbSaver::Impl {
   ~Impl();
```

Comment:
I didn’t understand what you mean. Impl is a final class.

Reply:
The fact that it's final doesn't mean the d'tor shouldn't be virtual.
If someone will `delete` a `SnapshotDataConsumer*` pointing to an `Impl` it will leak if it won't be virtual..

Reply:
@chakaz  but SnapshotDataConsumer's destructor is virtual. I *think* it's enough so that any derived class will have a virtual d'tor (correct me if I am wrong).

Reply:
Yes, you are right. If the parent (of any ancestor) is virtual that's enough. My bad.